### PR TITLE
BIP 176: Utilization of bits denomination

### PR DESCRIPTION
**Pull request by jimmysong**

### CODE REVIEW

**Comment by luke-jr** [comment]
For merge, rename to use BIP 176 and add to README.

Suggestions:

* Definition should probably be specified in satoshis
* Title might be better off something like "Bits denomination" or something

**Comment by jimmysong** [comment]
@luke-jr Suggestions addressed!

File: bip-0176.mediawiki [hunk 1]
```diff
@@ -0,0 +1,57 @@
+<pre>
+    BIP: 176
+    Title: Bits denomination
+    Author: Jimmy Song <jaejoon@gmail.com>
+    Comments-URI:  https://github.com/bitcoin/bips/wiki/Comments:BIP-????
```
Comment (luke-jr) [inline bip-0176.mediawiki]: Needs fixing

Comment (luke-jr) [reply bip-0176.mediawiki]: Need Comments-Summary too

File: bip-0176.mediawiki [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    Status: Draft
+    Type: Informational
+    Created: 2017-12-12
+    License: BSD-2-Clause
+    License-Code: BSD-2
```
Comment (luke-jr) [inline bip-0176.mediawiki]: There's no code in this?

**Comment by jimmysong** [comment]
Apologies for the premature squash. I think I've addressed your comments. @luke-jr

**Comment by jimmysong** [comment]
Got build to pass. Apologies for not checking that earlier.

**Comment by tamasblummer** [comment]
Amounts denominated in bits would fit nicely into legacy database structures and UIs with two decimal places for currency.

**Comment by sumBTC** [comment]
What about "coinbits"?

**Comment by luke-jr** [comment]
This isn't the right place to comment on the BIP.

https://lists.linuxfoundation.org/mailman/listinfo/bitcoin-dev

**Comment by tmornini** [comment]
I think "bit" is the **absolute worst** possible choice.

It's nearly impossible to choose a more overloaded word, particularly in the context of software and money:

4 etymologies with a whopping 7 definitions that are close enough that they could lead to confusion:

```
4 : the part of a key that enters the lock and acts on the bolt and tumblers (see tumbler 3a)
2 : to form a bit (see 1bit 4) on (a key)
2 a : a small piece or quantity of some material thing broken bits of glass picking bits of lint off her sweater
3 : something small or unimportant of its kind
1 : a unit of computer information equivalent to the result of a choice between two alternatives (such as yes or no, on or off)
2 : the physical representation of a bit by an electrical pulse, a magnetized spot, or a hole whose presence or absence indicates data
2 : b (1) British : a coin of a specified small denomination a threepenny bit (2) : a unit of value equal to ¹/₈ of a U.S. dollar (12 ¹/₂ cents) —used only of even multiples
```

https://www.merriam-webster.com/dictionary/bit

I'd prefer to go with a more metric theme along the lines of millis and micros.

**Comment by stefek99** [comment]
@luke-jr I've followed your advice about mailing list: 

> Detailed patch discussion generally better on a GitHub PR. 

Plus there is nothing on the list:

image


---

Bitcoin: `1 BTC`

Satoshi: `0.00000001 BTC`

The only other units I've ever heard in my life:
- millis
- bits

In my head 1 bit is `0.001` (one thousand of the BTC)

---

At the current rates 1 bit (according to my head) is `$13.5` - I can handle such denomination in my head:

- lunch is 1 bit
- bear is 0.3 bit
- plane ticket is 10 bits

--- 

My suggestion would be to revisit this BIP and PR and do a poll... I don't see much activity here, sample is too small.

**Comment by luke-jr** [comment]
That's the moderation ML, not the bitcoin-dev ML...

This PR is merged; it's a closed topic.

Feel free to do whatever polling you want, but BIPs are optional regardless.

**Comment by stefek99** [comment]
Ouch. Wrong list.

This is the correct link: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2017-December/015399.html

Github PRs are not set in stone. It is always possible to revisit the thing

---

Did Google search for "bitcoin bits" and you are probably right

* https://en.bitcoin.it/wiki/Units#Bits
* https://support.xapo.com/what-are-bits

> cos musialo mi sie pojebac

(a polish phrase)

**Comment by murchandamus** [comment]
Hi @jimmysong, I was just looking through the list of BIPs in Draft Status. It seems to me that BIP176 has not had any updates in a very long time, so I surmise that all planned work on it is complete. Would you mind if it were advanced to Complete?

**Comment by jimmysong** [comment]
@murchandamus that would be great. Thank you.

**Pull request merged**